Exception in createOutgoingCall()

I run into the following exception when trying to initiate a 1:1 Jingle A/V call with Jitsi 2.11.5602 on Debian 9.3 (stretch), while everything works fine when the contact calls me:

```
net.java.sip.communicator.service.protocol.OperationFailedException: Failed to create a call to juliet@example.com/phone
	at net.java.sip.communicator.impl.protocol.jabber.ProtocolProviderServiceJabberImpl.throwOperationFailedException(ProtocolProviderServiceJabberImpl.java:2857)
	at net.java.sip.communicator.impl.protocol.jabber.OperationSetBasicTelephonyJabberImpl.createOutgoingCall(OperationSetBasicTelephonyJabberImpl.java:467)
	at net.java.sip.communicator.impl.protocol.jabber.OperationSetBasicTelephonyJabberImpl.createOutgoingCall(OperationSetBasicTelephonyJabberImpl.java:286)
	at net.java.sip.communicator.impl.protocol.jabber.OperationSetBasicTelephonyJabberImpl.createOutgoingCall(OperationSetBasicTelephonyJabberImpl.java:255)
	at net.java.sip.communicator.impl.protocol.jabber.OperationSetVideoTelephonyJabberImpl.createOutgoingVideoCall(OperationSetVideoTelephonyJabberImpl.java:149)
	at net.java.sip.communicator.impl.protocol.jabber.OperationSetVideoTelephonyJabberImpl.createVideoCall(OperationSetVideoTelephonyJabberImpl.java:99)
	at net.java.sip.communicator.impl.gui.main.call.CallManager.internalCallVideo(CallManager.java:2660)
	at net.java.sip.communicator.impl.gui.main.call.CallManager.access$800(CallManager.java:60)
	at net.java.sip.communicator.impl.gui.main.call.CallManager$CreateCallThread.run(CallManager.java:2589)
Caused by: java.lang.NullPointerException
	at net.java.sip.communicator.impl.protocol.jabber.CallPeerJabberImpl.setState(CallPeerJabberImpl.java:1413)
	at net.java.sip.communicator.service.protocol.AbstractCallPeer.setState(AbstractCallPeer.java:931)
	at net.java.sip.communicator.service.protocol.AbstractCallPeer.setState(AbstractCallPeer.java:917)
	at net.java.sip.communicator.impl.protocol.jabber.CallJabberImpl.initiateSession(CallJabberImpl.java:788)
	at net.java.sip.communicator.impl.protocol.jabber.OperationSetBasicTelephonyJabberImpl.createOutgoingCall(OperationSetBasicTelephonyJabberImpl.java:449)
	... 7 more
```

--- a/src/net/java/sip/communicator/impl/protocol/jabber/CallPeerJabberImpl.java
+++ b/src/net/java/sip/communicator/impl/protocol/jabber/CallPeerJabberImpl.java
@@ -1410,7 +1410,18 @@ public class CallPeerJabberImpl
              */
             if (CallPeerState.DISCONNECTED.equals(newState)
                     || CallPeerState.FAILED.equals(newState))
-                getMediaHandler().getTransportManager().close();
+            {
+                CallPeerMediaHandlerJabberImpl mediaHandler = getMediaHandler();
+                if (mediaHandler != null)
+                {
+                    TransportManagerJabberImpl transportManager
+                        = mediaHandler.getTransportManager();
+                    if (transportManager != null)
+                    {
+                        transportManager.close();
+                    }
+                }
+            }
         }
         finally
         {

